src/modules/mc_pos_control/multicopter_position_mode_params.c

```diff
@@ -90,17 +90,16 @@ PARAM_DEFINE_FLOAT(MPC_VEL_MAN_SIDE, -1.0f);
  * @unit m/s
  * @min -1.0
  * @max 20.0
- * @increment 0.1
- * @decimal 2
+ * @increment 1
+ * @decimal 0
  * @group Multicopter Position Control
  */
 PARAM_DEFINE_FLOAT(MPC_VEL_MAN_BACK, -1.0f);
 
 /**
- * Maximum ascent velocity
+ * Maximum ascent velocity in manually piloted climb rate controlled modes and offboard
```

Comment:
Well, kind of. It's the absolute limit for maximum vertical velocity so actually it should be in `multicopter_position_control_limits_params.c`. That limit is for the generation only used in manually piloted modes. All autonomous ones use the lower `MPC_Z_V_AUTO_UP` and just get truncated by the controller to not exceed this value (usually not necessary). But there are exceptions if you're already faster than this velocity when switching into the climb rate controller mode. Let me move it to the limits because that's what it actually is.

Reply:
b34df9d9d822a6d473155bce4722d3785f398774

Reply:
perfect